frontend/src/components/atoms/elements/Image.tsx

```diff
@@ -1,101 +1,85 @@
 import { useState } from 'react';
-
 import Skeleton from '@mui/material/Skeleton';
+import Lightbox from 'yet-another-react-lightbox';
+import 'yet-another-react-lightbox/styles.css';
+import Zoom from 'yet-another-react-lightbox/plugins/zoom';
+import Download from 'yet-another-react-lightbox/plugins/download';
```

Comment:
That's a lot of dependencies to import, increasing binary size _even if the feature is not turned on_. How much larger is the output bundle -- and could we somehow make the lightbox optional in the frontend?

Perhaps we can set up stuff so it's a separate chunk, only importing the chunk in the browser if and when the lightbox feature is enabled?

Reply:
Ill look into it.